Sources/HttpPipeline/Middleware.swift

```diff
@@ -99,30 +106,24 @@ public func send(_ data: Data?) -> Middleware<BodyOpen, BodyOpen, Data?, Data?>
 }
 
 public func respond<A>(text: String) -> Middleware<HeadersOpen, ResponseEnded, A, Data?> {
-  return { conn in
-    conn.map(const(text.data(using: .utf8)))
-      |> writeHeader(.contentType(.text))
-      |> closeHeaders()
-      |> end
-  }
+  return map(const(text.data(using: .utf8)))
```

Comment:
was able to write some of helpers in point-free by pre-composing (i.e. `map` on functions) with a `map(const)` on `Conn`.